test.js

```diff
@@ -163,3 +165,114 @@ test(`use relative path with '..' chars`, async t => {
 	const {stdout} = await m(pathViaParentDir, ['foo']);
 	t.is(stdout, 'foo');
 });
+
+test('err.killed is true if process was killed directly', async t => {
+	const cp = m('forever');
+
+	setTimeout(function () {
+		cp.kill();
+	}, 100);
+
+	const err = await t.throws(cp);
+
+	t.true(err.killed);
+});
+
+// TODO: Should this really be the case, or should we improve on child_process?
+test('err.killed is false if process was killed indirectly', async t => {
```

Comment:
@sindresorhus - What do you think about this?

Right now, we have this behavior:

```js
cp.kill(); // err.killed will be true
process.kill(cp.pid); // err.killed will be false
```

This mimics the behavior of `child_process.exec`, but it doesn't make a lot of sense to me. If the process is killed, shouldn't `killed` be true regardless of how it was done?